src/app/shared/metadata-representation/metadata-representation.decorator.spec.ts

```diff
@@ -76,4 +96,44 @@ describe('MetadataRepresentation decorator function', () => {
       });
     });
   });
+
+  describe('With theme extensions', () => {
+    describe('If requested theme has no match', () => {
```

Comment:
Maybe I'm misunderstanding the theme extensions tests here...but, it seems like we are missing a test to prove extends is working?  I just see a test for "has no match" and "theme extension cycle".  Where's a test for a match?

UPDATE: It might be that this 'describe' is just badly named...it's just very hard to follow this test as-is without a bit more comments....I've had to try and review the beforeEach & the `it` clauses several times, and I'm still not sure it makes sense.

Reply:
The "there is a match" case is already covered by the existing tests; with these tests it should be enough to cover how we fall back to other themes if there is no match. I've added a little disclaimer comment.

Not sure how we can reword these`describe`s to make that more clear though...

Reply:
@ybnd : The renaming of variables and the `it` clauses helped make this a lot clearer. Thanks for that....my prior confusion was that the describe/it simply seemed contradictory, but with the updated language I see I had been misreading them. Thanks!